pkg/statistics/handle/syncload/stats_syncload.go

```diff
@@ -196,7 +197,10 @@ func (s *statsSyncLoad) SubLoadWorker(sctx sessionctx.Context, exit chan struct{
 			case errExit:
 				return
 			default:
-				time.Sleep(s.statsHandle.Lease() / 10)
+				// To avoid the thundering herd effect
```

Comment:
It seems that sync load tasks are processed sequentially here, and a single sync load task is not concurrent. So why does the thundering herd effect occur?

Reply:
```stats-load-concurrency``` controls the concurrency of sync load. at default, it will create 5 concurrency to work for it.

Reply:
so if we have a large tidb cluster, it is easy that many node have to retry at the same time.